### PR TITLE
Add drag and drop feature

### PR DESCRIPTION
**Pull request by elassis**
# What I have accomplished 🏆
* Set the basic structure of the project
* Set webpack
* Set the styling
* Set the drag and drop feature
# Built With ⚒️
* HTML
* CSS
* JAVASCRIPT
* WEBPACK

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,14 +1,11 @@
+/* eslint-disable */
```
Comment (carlos-ssh) [inline src/index.js]: - Kindly enable your Linters.

File: src/dragDrop.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+    container.addEventListener('drop', (
+
+    ) => {
+      interchange(dragItem, container.firstElementChild);
+      setLocalStorage();
+    });
+  });
+}
+/* eslint-disable */
```
Comment (carlos-ssh) [inline src/dragDrop.js]: - Kindly enable your Linters.

File: src/status.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    if (element.checked === true) {
+      inputs[element.index].classList.add('done');
+      chks[element.index].checked = true;
+    } else {
+      inputs[element.index].classList.remove('done');
+    }
+  });
+}
+function setLocalStorage() {
+  const arr = [];
```
Comment (carlos-ssh) [inline src/status.js:16]: - Kindly fix this error in chrome console: `` Uncaught TypeError: Cannot read property 'classList' of undefined ``.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @elassis , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Provide a License.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mimipeshy — APPROVED** [review]
Hi @elassis 👋🏾 👋🏾 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Pull request merged**